src/ECER.Managers.Registry.Contract/Communications/Contract.cs

```diff
@@ -22,6 +22,19 @@ public record UserCommunicationQuery : IRequest<CommunicationsQueryResults>
 }
 public record CommunicationsQueryResults(IEnumerable<Communication> Items);
 
+public record SendMessageCommand(Communication communication, string userId) : IRequest<SendMessageResult>;
+
+public class SendMessageResult
+{
+  public string? CommunicationId { get; set; }
```

Comment:
should CommunicationId always be present for a post?

Reply:
@SoLetsDev - in the SendMessageResult class -> is just for returning newly added communication(message) ID back as response...

Reply:
oh I get it, the result can be a successful or a failure... which might not have a communicationId